### ISSUE TITLE
Question about training dataset

### ISSUE THREAD

**Issue by piddnad** [KIMGEONUNG/BigColor#2]
Thanks for your great work!

I read the paper and found that the following sentence was mentioned in the experiment section.
"We train our model on 1.2M color images of the ImageNet 1K training set after **excluding 10% original images with low colorful- ness scores**."

Which pictures were removed from the original ImageNet training set?
I would appreciate it if you could share a list of deleted images or code to detect low-quality images.

**Comment by KIMGEONUNG** [comment]
Thank you for your interest on our research.

I added a file you want to get into this repository ([here](https://github.com/KIMGEONUNG/BigColor/blob/main/srcs/list_low_colorful.csv))

More specifically, we excluded the images with a less colorfulness score than 15, which is the lowest colorful category defined in a research "Measuring Colourfulness in Natural Images".

If you need the measurement code, refer [here](https://github.com/KIMGEONUNG/comars-core/blob/master/scripts/metric/colorfullness#L24-L38)

If you have any remaining question, please let me know.

**Comment by piddnad** [comment]
Your reply is very clear and answers all my questions. 

Thank you for your kindly reply!

**Comment by AlexanderKozhevin** [comment]
@piddnad sorry, were you able to run inference with pretrained model?
